src/polycubed/src/server/Resources/Endpoint/Resource.cpp

```diff
@@ -33,4 +46,152 @@ Operation Resource::OperationType(bool update, bool initialization) {
     }
   }
 }
+
+void Resource::UpdateCubesConfig(const std::string& resource,
+                                 nlohmann::json body,
+                                 Operation opType) {
+
+  std::stringstream ss(resource);
+  std::vector<std::string> resItem;
+  std::string token;
+
+  while(std::getline(ss, token, '/')) {
+    resItem.push_back(token);
+  }
+
+  const std::string &serviceName = resItem[0];
+  const std::string &cubeName = resItem[1];
+
+  std::lock_guard<std::mutex> lg(mutex);
+  switch (opType) {
```

Comment:
Add some comments presenting how the system works.
If I remember correctly, configuration is saved in memory (a map?), then periodically saved on disk by the detached thread. This must be documented at least in the code, otherwise it becomes difficult to follow.